packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -34,7 +34,30 @@ const Duration _kScrollbarTimeToFade = Duration(milliseconds: 600);
 /// [showTrackOnHover]. The thickness of the track and scrollbar thumb will
 /// become larger when hovering, unless overridden by [hoverThickness].
 ///
-// TODO(Piinks): Add code sample
+/// {@tool dartpad --template=stateless_widget_scaffold}
+///
+/// This sample shows [Scrollbar] for a fraction of seconds while scrolling.
+/// If you want to persist [Scrollbar] then set [isAlwaysShown] is true along with
+/// [ScrollController].
```

Comment:
```suggestion
/// This sample shows a [Scrollbar] that executes a fade animation as scrolling occurs.
/// The Scrollbar will fade into view as the user scrolls, and fade out when scrolling stops.
```

```
/// If you want to persist [Scrollbar] then set [isAlwaysShown] is true along with
/// [ScrollController].
```
This part you mentioned here ^, can be a separate example that actually shows this. :) Feel free to add it here.

Reply:
You could add a controller here and mentioned it is interactive too.
Or add another example that shows how it uses the primary scroll controller.

Reply:
sure @Piinks